ProcessLib/GroundwaterFlow/GroundwaterFlowFEM.h

```diff
@@ -156,7 +156,7 @@ class LocalAssemblerData
 
     std::vector<std::vector<double>> _darcy_velocities
         = std::vector<std::vector<double>>(
-            GlobalDim, std::vector<double>(ShapeFunction::NPOINTS));
+            GlobalDim, std::vector<double>(_integration_method.getNumberOfPoints()));
```

Comment:
Regarding the failing clang compiler (#1448): Maybe this code should be moved to the initializer list of the constructor.